CMakeLists.txt

```diff
@@ -78,9 +78,31 @@ if (BUILD_SDF)
   # available during build time
   set(GZ_TOOLS_VER 2)
 
+  #################################################
+  # Find python
+  if (SKIP_PYBIND11)
+    message(STATUS "SKIP_PYBIND11 set - disabling python bindings")
+    find_package(Python3 COMPONENTS Interpreter)
+  else()
+    find_package(Python3 COMPONENTS Interpreter Development
+    )
```

Comment:
Nitpick: single line for the find_package command

Reply:
👍 I'll change it

Reply:
https://github.com/gazebosim/sdformat/pull/1370/commits/c9340301f156b6dba64c94b5adb67f6f9a3a19c8